### DISCUSSION TITLE
2023-05-17-eyes

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#139] [General]
# 全飞秒近视手术过程和恢复记录 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Life5月中旬做了一个全飞秒近视手术，在这里记录一下当天、术后的感受，希望可以给到好奇的同学一些参考。

https://tw93.fun/2023-05-17/eyes.html

**Comment by marianasignal** [comment]
请问去了哪家医院啊？

**Reply by tw93** [reply]
抱歉 这个不太方便告知 因为会有推销嫌疑

**Reply by marianasignal** [reply]
谢谢  理解

**Comment by boxsuny** [comment]
https://v.flomoapp.com/mine/?memo_id=Njg1NjM1MzQ，一个小宇宙播客节目，内容关于是否找到合格的验光，是否推荐做眼镜手术。

**Reply by boxsuny** [reply]
https://www.xiaoyuzhoufm.com/episode/64656256ce89512c4c88c21a，下面这个。

**Comment by boxsuny** [comment]
算了，我放弃了，是播客：晚点聊35期节目。

**Reply by tw93** [reply]
哈哈哈 发现点击进去看不到

**Comment by ovws** [comment]
做完之后没有限制电子设备使用时常吗？

**Reply by tw93** [reply]
有的 当天其实是看不了的 到晚上慢慢ok 后面一个月都需要控制一下

**Comment by boxsuny** [comment]
您这个可以增加一个功能删除，自己的留言的，我本来想删除的，发现没有这个功能。我在试一试发一个链接我好像一直没有成功过🤣，https://www.xiaoyuzhoufm.com/episode/64656256ce89512c4c88c21a?s=eyJ1IjoiNWU5MWFjZWY1M2FkNjZkOGI1ZWZlNzdkIiwiZCI6MX0%3D

**Comment by aguige** [comment]
成都高新数字大厦那有一家新开的，华西邓教授在那亲自手术。

**Comment by TopJohn** [comment]
慌得一批，还是戴眼镜吧，博主勇气可嘉

**Comment by yueim** [comment]
**眼睛**等外部东西
SHOULD BE
**眼镜**等外部东西

如果是故意卖萌，请忽略

**Reply by yueim** [reply]
**眼镜**本身的健康状态
=>
**眼睛**本身的健康状态

**Reply by tw93** [reply]
谢谢 我的笔误 修复一下

**Comment by here-ted** [comment]
只用一天就可以恢复?我记得很久之前我同学说要一个月不能见光, 是科技进步了吗?

**Reply by tw93** [reply]
全飞秒第二天我就去爬山了🤣

**Comment by wind33-will** [comment]
关于手术当天第三段第一行中的 “不能带手机，眼睛等外部” 应该是手机“眼镜”把，哈哈哈

**Reply by tw93** [reply]
谢谢提醒，已经修复，看起来是当时眼睛还没有恢复好导致

**Comment by dengweiqiang** [comment]
感觉明白风险，有预期就好，我自己对这种未知还是不太敢尝试。

**Reply by tw93** [reply]
是的，这种事情很多时候建议去正规医院，多去些地方问问

**Comment by fuwu1245** [comment]
做之前眼镜多少度数？有散光不

**Reply by tw93** [reply]
200-300的样子，有轻微散光

**Comment by VmzXv** [comment]
上周四女票刚做完，半飞。恢复挺好，她说无炫光和干涩情况。
还在听医生的滴眼药水。度数200多和400多，轻微散光。
当天五点多出手术室，之后1-2小时候单只眼睛有疼痛情况，可以忍受。之后症状消失，估计是麻药刚过。江苏省中医，医生李凯

**Comment by l5ee** [comment]
做的全飞秒4.0吗还是pro。这几个好像还有区别有点纠结
